PWGHF/HFC/TableProducer/correlatorLcHadrons.cxx

```diff
@@ -323,10 +350,15 @@ struct HfCorrelatorLcHadrons {
     registry.add("hEtaMcGen", "Lc,Hadron particles - MC Gen", {HistType::kTH1F, {axisEta}});
     registry.add("hPhiMcGen", "Lc,Hadron particles - MC Gen", {HistType::kTH1F, {axisPhi}});
     registry.add("hMultFT0AMcGen", "Lc,Hadron multiplicity FT0A - MC Gen", {HistType::kTH1F, {axisMultiplicity}});
-
     corrBinning = {{binsZVtx, binsMultiplicity}, true};
   }
 
+  template <typename Coll>
+  float evaluateCentralityColl(const Coll& collision)
+  {
+    return o2::hf_centrality::getCentralityColl<Coll>(collision);
+  }
```

Comment:
This is needed only if you foresee to change the centrality estimator, otherwise you can simply use the `collision.centFT0M()`. Is this the case?